src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -78,7 +78,9 @@ QgsIdentifyResultsWebView::QgsIdentifyResultsWebView( QWidget *parent ) : QgsWeb
   setSizePolicy( QSizePolicy::MinimumExpanding, QSizePolicy::Minimum );
   page()->setNetworkAccessManager( QgsNetworkAccessManager::instance() );
   // page()->setLinkDelegationPolicy( QWebPage::DelegateAllLinks );
+#ifdef WITH_QTWEBKIT
```

Comment:
The Homebrew `qt5` formula does have the `--with-qtwebkit` option, but this requires source building of `qt5` and probably `pyqt5` as well (though the latter does _not_ have a `--with-qtwebkit` option, so must opportunistically generate the bindings it QWebKit is available). While this is OK for working with general builds, it would take way too long for Travis CI.

Another option might be to host a `qt5-webkit` formula that builds against an installed `qt5`, though I have no idea if the CMake and PyQt5 setups will work right with the module built _outside_ of the Qt5 build chain. Worth trying, as it would be the simplest solution.

Yet another option is to do a PR for Homebrew's `qt5` formula to include QWebKit as default now with 5.7+ versions.